build.sh

```diff
@@ -33,20 +33,16 @@ usage() {
 # ==============================================================================
 
 # Defaults:
-INSTALL_PREFIX=""
 CMAKE_OPTS=""
 BUILD_TARGET="${MACHINE_ID:-'localhost'}"
 BUILD_VERBOSE="NO"
 CLONE_JCSDADATA="NO"
 CLEAN_BUILD="NO"
-BUILD_JCSDA="NO"
 COMPILER="${COMPILER:-intel}"
+WORKFLOW_BUILD=${WORKFLOW_BUILD:-"OFF"}
 
 while getopts "p:t:c:hvdfa" opt; do
   case $opt in
-    p)
```

Comment:
This should be retained.
The default could be `${dir_root}` or the user could pass `-p ${HOMEgfs}`